### ISSUE TITLE
Throw exception when I do LDA

### ISSUE THREAD

**Issue by YihengJiang** [kaldi-asr/kaldi#2662]
Dear professor：
My experiments are speaker recognition. I do LDA after extracted vectors, I use the way like x-vector but change the neural network. It's unfortune that I get an error as follow(excerpts from lda's log file):

# Running on cpu48
# Started at Sat Sep 1 19:37:11 CST 2018
# ivector-compute-lda --total-covariance-factor=0.0 --dim=128 "ark:ivector-subtract-global-mean ark:/home/jyh/D/jyh/data/fea_sitw_dl/exp/pytorch/train.ark ark:- |" ark:/home/jyh/D/jyh/data/fea_sitw_dl/data/train_combined_200k/utt2spk /home/jyh/D/jyh/data/fea_sitw_dl/exp/xvector_nnet_1a/xvectors_train_combined_200k/transform.mat 
ivector-compute-lda --total-covariance-factor=0.0 --dim=128 'ark:ivector-subtract-global-mean ark:/home/jyh/D/jyh/data/fea_sitw_dl/exp/pytorch/train.ark ark:- |' ark:/home/jyh/D/jyh/data/fea_sitw_dl/data/train_combined_200k/utt2spk /home/jyh/D/jyh/data/fea_sitw_dl/exp/xvector_nnet_1a/xvectors_train_combined_200k/transform.mat 
ivector-subtract-global-mean ark:/home/jyh/D/jyh/data/fea_sitw_dl/exp/pytorch/train.ark ark:- 
LOG (ivector-subtract-global-mean[5.4.185~1-2fa7]:main():ivector-subtract-global-mean.cc:76) Read 200000 iVectors.
LOG (ivector-subtract-global-mean[5.4.185~1-2fa7]:main():ivector-subtract-global-mean.cc:79) Norm of iVector mean was 2.13583
LOG (ivector-subtract-global-mean[5.4.185~1-2fa7]:main():ivector-subtract-global-mean.cc:108) Wrote 200000 mean-subtracted iVectors
LOG (ivector-compute-lda[5.4.185~1-2fa7]:main():ivector-compute-lda.cc:267) Read 200000 utterances, 0 with errors.
LOG (ivector-compute-lda[5.4.185~1-2fa7]:main():ivector-compute-lda.cc:273) Computing within-class covariance.
LOG (ivector-compute-lda[5.4.185~1-2fa7]:main():ivector-compute-lda.cc:278) 2-norm of iVector mean is 5.94342e-08
LOG (ivector-compute-lda[5.4.185~1-2fa7]:ComputeLdaTransform():ivector-compute-lda.cc:122) Stats have 6430 speakers, 200000 utterances. 
### WARNING (ivector-compute-lda[5.4.185~1-2fa7]:Cholesky():tp-matrix.cc:110) Cholesky decomposition failed. Maybe matrix is not positive definite. Throwing error
Cholesky decomposition failed.# Accounting: time=24 threads=1
# Finished at Sat Sep 1 19:37:35 CST 2018 with status 255

I try to understand Cholesky decomposition,and consider where is wrong,but it has no effective. So I want to ask for some help.
Now,I have an idea for this problem is that the training of my network did not sufficient. Cause I just train one epoch(84 egs file). In this case, I train network again(but result has not came out, now), at the mean time, I put this issue to you for some help. Dear professor, I want to know if it is the point of this problem, or it has another reason? Hope response,thank you!!!
